gapis/api/vulkan/frame_loop.go

```diff
@@ -318,6 +318,12 @@ func newFrameLoop(ctx context.Context, graphicsCapture *capture.GraphicsCapture,
 
 func (f *frameLoop) Transform(ctx context.Context, cmdId api.CmdID, cmd api.Cmd, out transform.Writer) {
 
+	// If we're looping only once we can just passthrough commands
+	if f.loopCount == 1 {
```

Comment:
This check has been done in gapis/api/vulkan/replay.go:927. And if want to make sure, maybe we want to check `if f.loopCount <= 1 `

Reply:
There is code handling f.loopCount == 0 elsewhere in this file :)

As for having two checks, I think this is fine. The transform should behave intelligently with any representable value passed in as it's dangerous and irresponsible to do otherwise. But there is also no reason to create the transform in the chain if it is of no benefit at the point of instantiation.